Consul.Test/Consul.Test.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.1;netcoreapp3.1;net461</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net461;$(TargetFrameworks)</TargetFrameworks>
```

Comment:
hm didn't you mention that `net461` could be built everywhere? If so it should probably run everywhere as well?

Reply:
Maybe we don't want that tho

Reply:
It can be built everywhere but not there is a problem with running all tests successfully on non-Windows platforms (most of them are passing just fine though).